src/domain/app/App.tsx

```diff
@@ -34,8 +36,13 @@ class App extends React.Component<AppProps> {
           this.props.fetchApiTokenError({ message: 'No user found' });
         }
       })
-      .catch(error => this.props.fetchApiTokenError(error));
+      .catch(error => {
+        this.props.fetchApiTokenError(error);
```

Comment:
We should clear oidc localStorage here i think

Reply:
At some point, adding a TODO for now